motoko/fileupload/package-lock.json

```diff
@@ -0,0 +1,8329 @@
+{
```

Comment:
Should `package-lock.json` be checked in. Is it for the other examples? Perhaps @krpeacock or @chenyan-dfinity can advise

Reply:
package-lock can be checked